src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/HistoricalQuestion.kt

```diff
@@ -26,7 +28,7 @@ class HistoricalQuestion(
 
   val additionalInformation: String? = null,
 
-  @OneToMany(mappedBy = "historicalQuestion", fetch = FetchType.EAGER, cascade = [CascadeType.ALL], orphanRemoval = true)
+  @OneToMany(mappedBy = "historicalQuestion", fetch = FetchType.LAZY, cascade = [CascadeType.ALL], orphanRemoval = true)
```

Comment:
actually, it's the other way round! we used to have the lazy type annotation and that spawned a HUGE number of sql statements. changing to eager helped… ref #97

Reply:
For one class with only one 1:M it might be ok, but its always better to explicity fetch not by default.